test/e2e/features/basic.feature

```diff
@@ -148,7 +148,7 @@ Feature: Basic test
         # This will remove the pull secret from the instance and from the cluster
         # You need to provide pull secret file again if you want to start this cluster
         # from a stopped state.
-        Given executing "crc bundle generate -f" succeeds
+        Given executing "crc bundle generate -f --log-level debug" succeeds
```

Comment:
Fwiw, rather than patching each crc execution we want debug for, we could take advantage of ` test/extended/crc/cmd/cmd.go` to force it for all `crc` use in the test suite, something like https://github.com/cfergeau/crc/commit/678cf7d909aae5c7a107d8dc0abf0857cfeed8e0 in combination with https://github.com/cfergeau/crc/commit/93c52920073aa13dcc573836dab0ee5910000ee3